plugins/media/wowza/services/LiveConversionProfileService.php

```diff
@@ -285,13 +286,20 @@ protected function appendLiveParams(LiveStreamEntry $entry, WowzaMediaServerNode
 		$keyFrameInterval->addChild('FollowSource', 'true');
 		$keyFrameInterval->addChild('Interval', 60);
 
-		$skipFrameCountPos = strpos($liveParams->getConversionEnginesExtraParams(), 'SkipFrameCount');
-		if ($skipFrameCountPos !== false)
-		{
-			$skipFrameCount = $video->addChild('SkipFrameCount');
-			preg_match('/SkipFrameCount=(\d+)/', $liveParams->getConversionEnginesExtraParams(), $skipFrameValue, $skipFrameCountPos);
-			$skipFrameCount->addChild('Value', (int)$skipFrameValue[1]);
-		}
+        if ($conversionExtraParam !== null && $conversionExtraParam->skipFrameCount !== null)
+        {
+            $skipFrameCount = $video->addChild('SkipFrameCount');
+            $skipFrameCount->addChild('Value', (int)$conversionExtraParam->skipFrameCount->value);
+        }
+
+		if ($conversionExtraParam !== null && $conversionExtraParam->constantBitrate !== null)
+        {
+            $parameters = $video->addChild('Parameters');
+            $parameter = $parameters->addChild('Parameter');
+            $parameter->addChild('Name', 'mainconcept.bit_rate_mode');
+            $parameter->addChild('Value', 0);
```

Comment:
Why not use the value of constantBitrate as we discussed this value represents a Wowza enum.